server/catgenome/src/main/java/com/epam/catgenome/controller/blast/BlastDataBaseController.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.catgenome.controller.blast;
+
+import com.epam.catgenome.controller.AbstractRESTController;
+import com.epam.catgenome.controller.Result;
+import com.epam.catgenome.entity.blast.BlastDataBase;
+import com.epam.catgenome.entity.blast.BlastDataBaseType;
+import com.epam.catgenome.exception.FeatureIndexException;
+import com.epam.catgenome.manager.blast.BlastDataBaseSecurityService;
+import com.wordnik.swagger.annotations.Api;
+import com.wordnik.swagger.annotations.ApiOperation;
+import com.wordnik.swagger.annotations.ApiResponse;
+import com.wordnik.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@Api(value = "blast-database", description = "BLAST Data Bases Management")
+@RequiredArgsConstructor
+public class BlastDataBaseController extends AbstractRESTController {
```

Comment:
I think it is more correct to use `Database` notation in camel case rather than `DataBase` or `Data Base`, could you please check all related classes?